packages/training/python/tests/test_enhanced_rewards_extended.py

```
@@ -0,0 +1,1130 @@
+    def test_regime_awareness_changes_score(self):
+        """Regime awareness should change score vs vanilla archetype."""
+        inputs = TrajectoryRewardInputs(
+            final_pnl=300,
+            starting_balance=10000,
+            end_balance=10300,
+            format_score=0.7,
+            reasoning_score=0.6,
+        )
+        
+        # Vanilla archetype reward
+        archetype = archetype_composite_reward(inputs=inputs, archetype="trader")
```

Ruff (0.14.11)

Local variable `archetype` is assigned to but never used

Remove assignment to unused variable `archetype`

(F841)